src/ScoreBoard.js

```diff
@@ -31,22 +34,23 @@ class ScoreBoard extends React.Component {
   }
 
   renderScoreboard() {
-    const { userMoves } = this.state;
+    const { userMoves, boardSize } = this.state;
     return (
       <section className="scoreboard">
         <div className="scoreboard-title">Score Board</div>
         <ol className="scoreboard-list">
-            {userMoves.map(({ name, moves }, index) => {
-              return (
-                <li
-                  key={index}
-                  className="scoreboard-item"
-                >
-                  <strong>{name}</strong>
-                  <span>: {moves} moves</span>
-                </li>
-              );
-            })}
+          {userMoves.map(({ name, moves }, index) => {
+            const finalMoves = moves * (BOARD_SIZE_MAX + 1 - boardSize);
```

Comment:
Instead of calculating `finalMoves` before rendering in list, calculate it before saving in `localStorage` [here](https://github.com/itaditya/trick-or-treat-game/blob/master/src/Game.js#L52). and also render the finalMoves in the result page [here](https://github.com/itaditya/trick-or-treat-game/blob/master/src/Game.js#L237).

Reply:
Saving as a separate variable, `adjustedMoves`.